lighthouse-core/lib/asset-saver.js

```diff
@@ -198,9 +198,29 @@ function saveAssets(artifacts, audits, pathWithBasename) {
   });
 }
 
+/**
+ * Log trace(s) and associated screenshot(s) to console.
+ * @param {!Artifacts} artifacts
+ * @param {!Audits} audits
+ * @return {!Promise}
+ */
+function logAssets(artifacts, audits) {
+  return prepareAssets(artifacts, audits).then(assets => {
+    const data = assets[0];
+    log.log('devtoolslog.json', JSON.stringify(data.devtoolsLog, null, 2));
+    log.log('screenshots.html', data.screenshotsHTML);
```

Comment:
this hurts a little, haha, do we want to perpetuate the screenshots html everywhere? if they don't need it I'd vote to leave out and remove the file entirely in v3 :)